wire-ios/Wire-iOS/Sources/Authentication/Coordinator/AuthenticationCoordinator.swift

```diff
@@ -901,6 +901,7 @@ extension AuthenticationCoordinator {
     }
 
     private func completeE2EIdentityEnrollment() {
+        executeActions([.showLoadingView])
         guard let session = statusProvider.sharedUserSession else { return }
```

Comment:
question: should we hide if we have no session?

Reply:
I mean either way you're stuck so I don't think it makes a difference